source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.js

```diff
@@ -214,19 +232,25 @@ export default class ChangeSpendingPasswordDialog extends Component<Props> {
 
     const newPasswordClasses = classnames(['newPassword', styles.newPassword]);
 
+    const currentPasswordField = form.$('currentPassword');
+    const newPasswordField = form.$('spendingPassword');
+    const repeatedPasswordField = form.$('repeatPassword');
+
+    const canSubmit = !isSubmitting && form.isValid;
+
+    const currentPasswordError =
+      canSubmit && error && error.code === 'wrong_encryption_passphrase';
```

Comment:
@DeeJayElly This is not necessary because you will return boolean and that means nothing here 
https://github.com/input-output-hk/daedalus/pull/1998/files#diff-a86097ad3cb7f7bffb9c0a3da3d6fdc3R282

Reply:
Removed